iio/iio_demo/iio_demo.c

```diff
@@ -452,7 +452,7 @@ int32_t iio_demo_init(struct iio_demo_desc **desc,
 		.write_data = iio_demo_write_dev,
```

Comment:
Check commit message, it doesn't sound right.

Reply:
done